prometheus-to-sd/translator/translator.go

```diff
@@ -301,6 +301,8 @@ func translateOne(config *config.CommonConfig,
 	}
 	setValue(mType, valueType, metric, point)
 
+	mr := getMonitoredResourceFromLabels(config, metric.GetLabel())
+	glog.Infof("MonitoredResource to write: %c", mr)
```

Comment:
V(4)? I don't think it makes sense to print this log line except for debugging. It's going to be quite chatty.

Reply:
Can you push your changes? It still looks the same on GitHub.

Reply:
Done. PTAL?